src/sentry/static/sentry/app/plugins/components/issueActions.tsx

```diff
@@ -299,15 +372,17 @@ class IssueActions extends PluginComponentBase {
 
     if (impactedField) {
       //if every dependent field is set, then search
-      if (!impactedField.depends.some(dependentField => !formData[dependentField])) {
+      if (!impactedField.depends?.some(dependentField => !formData[dependentField])) {
         callback = () => this.loadOptionsForDependentField(impactedField);
       } else {
         //otherwise reset the options
         callback = () => this.resetOptionsOfDependentField(impactedField);
       }
     }
-
-    this.setState({[formDataKey]: formData}, callback);
+    const newState = {[formDataKey]: formData} as {
+      [x in typeof formDataKey]: Record<string, any>;
+    };
```

Comment:
i ended up playing around with what's happening here and didn't find much other than maybe 
`Record<typeof formDataKey, Record<string, any>>` but our formatter makes it harder to read than the current solution